src/main/docker/kafdrop.sh

```diff
@@ -40,7 +40,7 @@ if [ "$KAFKA_PROPERTIES" != "" ]; then
   echo Writing Kafka properties into $KAFKA_PROPERTIES_FILE
   echo "$KAFKA_PROPERTIES" | base64 --decode --ignore-garbage > $KAFKA_PROPERTIES_FILE
 else
-  rm $KAFKA_PROPERTIES_FILE |& > /dev/null | true
+  if [ ! -f $KAFKA_PROPERTIES_FILE ]; then touch $KAFKA_PROPERTIES_FILE; fi
```

Comment:
Please merge this line with the one above like this:
```suggestion
  elif [ ! -f $KAFKA_PROPERTIES_FILE ]; then
    touch $KAFKA_PROPERTIES_FILE;
```

Can you apply this pattern for the keystore and the truststore too and update the README accordingly? That basically makes this PR equal to #242 (which I now see for the first time), except for the added documentation.

It will also enable running Kafdrop with secrets in Docker compose or K8s.

Reply:
Thanks, updated in [5ffb777](https://github.com/obsidiandynamics/kafdrop/pull/502/commits/5ffb777f027c78fd23450908941da5d2fbeecee9)